.ci/test.sh

```diff
@@ -14,7 +14,7 @@ function usage {
 
 if [[ "$COMMAND" == "install" ]]; then
     conda install --quiet matplotlib mkl numpy scipy
-    pip install coverage pytest<4
```

Comment:
Thanks. I was just encountering this in another branch.

Strangely, I upgraded my pytest to 4 on my machine, but I can't seem to reproduce it. So it's something a bit more particular about TravisCI.